requests_cache/response.py

```diff
@@ -119,6 +120,18 @@ def revalidate(self, expire_after: ExpirationTime) -> bool:
         self.expires = self._get_expiration_datetime(expire_after)
         return self.is_expired
 
+    @property
```

Comment:
Perhaps also `@cached_property`, so it would be calculated only once.

Here's py3.6-3.9 compatible `@memoize`:
- https://github.com/Taxel/PlexTraktSync/blob/0.8.16/plex_trakt_sync/decorators/memoize.py

Caching becomes relevant if the requests list is sorted pefire printing. I believe already the sorter evaluates the value multiple times.

Reply:
If you sorted on `response.size`, yes, it would get evaluated twice per response. But if you want to sort by size, wouldn't you want to use the length in bytes (`len(response.content)`) instead of a string?

And if you want that to be fast, doing the ordering in a SQL query is going to be the way to go.

Reply:
Yes, need unformatted value too. Maybe format in str method only? Not have separate method for formatted value.

Reply:
Yes, sorting and limiting supported by this project would be awesome.

I planned to ask this, but then deleted the comment as it would be requesting too many things at once.

It should be handed as separate feature request anyway.

Reply:
Sure, let's discuss that in a separate issue.